pom.xml

```diff
@@ -119,13 +119,77 @@
                                 <version>8.29</version>
                             </dependency>
                         </dependencies>
-                        <configuration>
+                        <configuration>     
```

Comment:
Please, restore the original formatting.
```suggestion
                        <configuration>
```

Reply:
resolved